apps/test/unit/netsim/NetSimRouterLogModal.js

```diff
@@ -123,6 +125,20 @@ describe("NetSimRouterLogModal", function () {
       modal.setRouter(null); // Normally netsim.js does this
       assert.equal('none', modal.currentTrafficFilter_);
     });
+
+    it("can render the dropdown with no local address", function () {
+      expect(() => {
+        modal.onShow_();
+      }).not.to.throw();
```

Comment:
This was passing due to the early-out in rendering the dropdown causing the issue.